content/developers.html

```diff
@@ -256,30 +278,25 @@ <h3>Further Reading</h3>
 
 <div class="panel-heading">
                 
-<h3 class="panel-title">Git Repositories</h3>
+<h3 class="panel-title">CloudStack Git Repositories</h3>
               
 </div>
               
 <div class="panel-body">
                 
-<p>The git repositories are hosted on Apache infrastructure, and can be found here:</p>
+<p>The git repositories are hosted on GitHub infrastructure, and can be found here:</p>
```

Comment:
no, this is not where it is hosted. This is where it is synced to. it is hosted at gitbox.apache.org

Reply:
@DaanHoogland returned back.